backend/routes/auth/authorize.py

```diff
@@ -21,7 +27,47 @@ class AuthorizeRequest(BaseModel):
 
 
 class AuthorizeResponse(BaseModel):
-    token: str = Field(description="A JWT token containing the user information")
+    username: str = Field("Discord display name.")
+    expiry: str = Field("ISO formatted timestamp of expiry.")
+
+
+AUTH_FAILURE = JSONResponse({"error": "auth_failure"}, status_code=400)
+
+
+async def process_token(bearer_token: dict) -> Union[AuthorizeResponse, AUTH_FAILURE]:
+    """Post a bearer token to Discord, and return a JWT and username."""
+    interaction_start = datetime.datetime.now()
+
+    try:
+        user_details = await fetch_user_details(bearer_token["access_token"])
+    except httpx.HTTPStatusError:
+        AUTH_FAILURE.delete_cookie("BackendToken")
```

Comment:
Same as the other note.
```suggestion
        AUTH_FAILURE.delete_cookie("token")
```